canonical/ubuntu.com
# fixing third level nav by moving the active class and adding one to t…

## Done

it seemed that in all 2nd level nav and in 3rd level in cloud the active is put on the &lt;a>; however, in download, desktop and management on 3rd its on the &lt;li> instead

I have moved it and added a 'breadcrumb-link--second-level' link as well to fix something I was trying to do with css

I can now remove a bunch of css from https://github.com/ubuntudesign/www.ubuntu.com/pull/507
## QA
1. go to /download/desktop/create-a-usb-stick-on-windows and see that the current page link is active in the nav
   DEMO - good
   ![image](https://cloud.githubusercontent.com/assets/441217/14666017/e315a950-06ce-11e6-9037-1ebf0134aae7.png)

LIVE - bad
![image](https://cloud.githubusercontent.com/assets/441217/14666030/fbea4ba2-06ce-11e6-81bc-90742aa82f42.png)
1. go to /cloud/openstack and see that there is a '>' between OpenStack and the 3rd level items

DEMO - good
![image](https://cloud.githubusercontent.com/assets/441217/14666079/37577106-06cf-11e6-95b1-e68cf0baabbb.png)

LIVE - bad
![image](https://cloud.githubusercontent.com/assets/441217/14666074/2b88197a-06cf-11e6-9291-1a5d687d45af.png)
## Issue / Card

Fixes #510, #505, #506
